@stellar/lyra-api/package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
`lyra-api` is the sub-repo that contains an npm package that exports all the helper functions needed by a client